UPGRADE-2.8.md

```diff
@@ -4,6 +4,37 @@
 Form
 ----
 
+ * The "checkbox" form-type now interprets '0' and '' values as unchecked unless you 
+   explicitly specified one of thoose as checked-value (the "value"-option). 
+   Any other value will transform to NULL.
+   
+   Before:
+   ```
+   true  <---> '1' (default checked-value)
+   false <---> null
+   (any other value were true as well)
+   true  <--- ''
+   true  <--- '0'
+   true  <--- 'foobar'
+   ```
+   
+   After:
+   ```
+   true  <---> '1' (default checked-value)
+   false <---> null
+   false <--- ''
+   false <--- '0'
+   (any other value will be NULL)
```

Comment:
right, but from the `CheckboxType` point of view it'll be `null`. maybe something like "any other value will internaly cause an `TransformationFailedException` which leads to a `null` form-value" ?? .... sry, i'm not good at writing prose ;)